tests/test_summary.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import sys
+import pytest
+import pandas as pd
+from mock import patch
+sys.path.append('../')
+sys.path.append('../webviz_subsurface/containers/')
+with patch('webviz_config.common_cache.cache.memoize',
+           lambda *x, **y: lambda f: f):
+    from webviz_subsurface.datainput import \
+        load_ensemble_set, \
+        get_time_series_statistics, \
+        get_time_series_fielgains, \
+        get_time_series_data
+
+
+# define recurring variables
+VOVLE_ENSEMBLE_PATHS = [
+    ('iter--0', '/scratch/fmu/stcr/volve/realization-*/iter-0'),
+    ('iter--1', '/scratch/fmu/stcr/volve/realization-*/iter-1'),
+    ('iter--2', '/scratch/fmu/stcr/volve/realization-*/iter-2'),
+]
```

Comment:
These hardcoded paths should probably not be there?

Reply:
No thats true. Where would we store some testing data? Or could be setup some config for testing? We would need some ensembles with know results to test if the functions return the desired output.